<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyBoss</class>
 <widget class="QDialog" name="MyBoss">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>465</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>老板界面</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QPushButton" name="loginAgain">
     <property name="text">
      <string>重新登录</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <widget class="QWidget" name="toolBoxPage1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>679</width>
        <height>366</height>
       </rect>
      </property>
      <attribute name="label">
       <string>账单</string>
      </attribute>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>391</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>209</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>92</width>
         <height>167</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="weeklyBill">
          <property name="text">
           <string>周结账单</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="monthlyBill">
          <property name="text">
           <string>月结账单</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>年</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>月</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="yearBill">
          <property name="text">
           <string>年结账单</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QComboBox" name="comboBox_3">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>年</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="toolBoxPage2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>679</width>
        <height>366</height>
       </rect>
      </property>
      <attribute name="label">
       <string>优惠</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>261</width>
         <height>201</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>已有优惠：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="kind">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>20</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>满100减10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>满200减20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>满300减30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>七折</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>八折</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>九折</string>
        </property>
       </item>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>244</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="addDiscount">
          <property name="text">
           <string>添加优惠</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteDiscount">
          <property name="text">
           <string>删除优惠</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>显示所有优惠</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>37</y>
         <width>203</width>
         <height>132</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>优惠id：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="id">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>优惠名称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>优惠种类：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_4">
            <item>
             <property name="text">
              <string>满100减10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>满200减20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>满300减30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>七折</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>八折</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>九折</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>起始时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="start"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>结束时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="end"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
